scripts/coderabbit_retry/tests/test_coderabbit_retry.py

```
@@ -0,0 +1,361 @@
+    def test_skips_wip_prs(self, mock_github_cls: MagicMock, mock_list: MagicMock, mock_process: MagicMock) -> None:
+        mock_list.return_value = [make_pr_issue(number=1, title="[WIP] my feature")]
+        assert main() == 0
+        mock_process.assert_not_called()
+
+    @patch("scripts.coderabbit_retry.coderabbit_retry.process_pr", side_effect=[True, False, True])
+    @patch("scripts.coderabbit_retry.coderabbit_retry.list_eligible_prs")
+    @patch("scripts.coderabbit_retry.coderabbit_retry.Github")
+    @patch.dict("os.environ", {"REPO": "owner/repo", "GH_TOKEN": "fake-token"}, clear=True)
+    def test_aggregates_results_from_multiple_prs(
+        self,
+        mock_github_cls: MagicMock,
```

Ruff (0.16.1)

[warning] Unused method argument: `mock_github_cls`

(ARG002)